src/gobu/main.go

```diff
@@ -113,7 +118,14 @@ func main() {
 	if arch == "arm" {
 		arch = "armv6l"
 	}
-	onlinePath = fmt.Sprintf(onlinePath, globalVersion, runtime.GOOS, arch)
+	
+	extension := "tar.gz"
```

Comment:
Use package var to avoid setting in multiple locations, or pass it as input to the 'download' func.